pkg/controllers/report/utils/scanner.go

```diff
@@ -261,23 +263,67 @@ func (s *scanner) ScanResource(
 				false,
 				nil,
 			)
-			engineResponse, err := engine.Handle(ctx, request, nil)
+			object, oldObject, _ := admissionutils.ExtractResources(nil, request.Request)
+
+			attr := admission.NewAttributesRecord(
+				&object,
+				&oldObject,
+				schema.GroupVersionKind(request.Request.Kind),
+				request.Request.Namespace,
+				request.Request.Name,
+				schema.GroupVersionResource(request.Request.Resource),
+				request.Request.SubResource,
+				admission.Operation(request.Request.Operation),
+				nil,
+				false,
+				nil,
+			)
+
+			targetMpols := provider.MatchesMutateExisting(ctx, attr, nil)
+			mpolMap := make(map[string]policiesv1alpha1.MutatingPolicy)
+			for _, p := range mpols {
+				mpolMap[p.GetName()] = *p.AsMutatingPolicy()
+			}
+
+			enabledNames := make([]string, 0)
+			for _, name := range targetMpols {
+				if p, ok := mpolMap[name]; ok && p.Spec.MutateExistingEnabled() {
+					enabledNames = append(enabledNames, name)
+				}
+			}
+
+			predicate := mpolengine.MatchNames(enabledNames...)
+
+			engineResponse, err := engine.Handle(ctx, request, predicate)
+
+			new := engineResponse.PatchedResource
+			if new != nil {
+				s.client.UpdateResource(ctx, new.GetAPIVersion(), new.GetKind(), new.GetNamespace(), new.Object, false, "")
```

Comment:
Does this PR mutate existing resources from background scan, or it enable reporting only?

The [issue](https://github.com/kyverno/kyverno/issues/13678) states reporting only "Enable background reporting for mutateExisting in mpol.".